package-lock.json

```diff
@@ -394,8 +393,7 @@
           "version": "4.4.0",
           "resolved": "https://registry.npmjs.org/file-type/-/file-type-4.4.0.tgz",
           "integrity": "sha1-G2AOX8ofvcboDApwxxyNul95BsU=",
-          "dev": true,
-          "optional": true
```

Comment:
Just to check--are you running NPM install from the `package.json` on master? Our current lock file may be out of date, so these changes are a nice bonus, but I want to understand why `"optional": true` was marked in so many places.

Reply:
eh i dont know why this change is present :joy:

Reply:
Possibly from running `make develop`? anyhow, ill revert. my bad